test/test-ipld-dag-cbor.js

```diff
@@ -0,0 +1,205 @@
+/* eslint-env mocha */
+'use strict'
```

Comment:
the tests in `cbor` and `proto` are super similar, maybe you could extract the test methods, and just pass in the type agains it should test?